readme.md

```diff
@@ -48,11 +48,13 @@ This is needed to be able to compare the current files with the destination file
 
 Can also be a function returning a destination directory path.
 
+If destination is `null` or `undefined`, only the source file SHA1 history will be used to determine if the file has changed or not. Upon the first run, all files will be treated as changed.
+
 #### options
 
 ##### cwd
 
-Type: `string`  
+Type: `string`
```

Comment:
Tip: use the `EditorConfig` plugin :). https://github.com/sindresorhus/gulp-changed/blob/master/.editorconfig#L14-L15.